release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -334,12 +347,20 @@ In the table, features are marked with the following statuses:
 |DEP
 |DEP
 
+|Scheduler policy
+|GA
+|GA
+|DEP
+
 |====
 
 [id="ocp-4-7-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-7-scheduler-policy-deprecated"]
+==== Scheduler policy
 
+Using a scheduler policy to control pod placement is deprecated and might be removed in a future release. For more information on the Technology Preview alternative, see xref:../nodes/scheduling/nodes-scheduler-profiles.adoc#nodes-scheduler-profiles[Scheduling pods using a scheduler profile].
```

Comment:
> might be removed

this will definitely be removed in a future release

Reply:
Yeah, I'll check on this. Usually we try to avoid predicting future things because they might change (and there could be legal issues with claiming something that doesn't happen). But I see that some of our other release notes entries say "will". :woman_shrugging: I'll double check.

Reply:
I assumed that there was a logical reason like that :) That makes sense

Reply:
A little wordy, but I went with "is planned for removal in a future release" because that more direct than "might" but leaves room for change it since it's only "planned". Semantics :)

Reply:
Sounds good to me! Thanks